io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/pubchem-align3d
+      - cp -r * /app/tmp/pubchem-align3d
+
+  - name: rapidjson
+    sources:
+      - type: git
+        url: https://github.com/Tencent/rapidjson
+        commit: 858451e5b7d1c56cf8f6d58f88cf958351837e53
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/rapidjson-1.1.0
+      - cp -r * /app/tmp/rapidjson-1.1.0
+
+  - name: rdkit
+    sources:
+      - type: archive
+        url: https://github.com/rdkit/rdkit/archive/refs/tags/Release_2023_09_4.tar.gz
+        sha256: abacae431bbc5882b87cc8629b7ddc02757204e854aa45b6157ec2bf45d623ef
+    buildsystem: simple
+    build-commands:
+      - cp -r /app/tmp/RingDecomposerLib External/RingFamilies/
+      - cp -r /app/tmp/pubchem-align3d External/pubchem_shape/
+      - cp -r /app/tmp/maeparser External/CoordGen/
+      - cp -r /app/tmp/coordgen External/CoordGen/
+      - cp -r /app/tmp/rapidjson-1.1.0 External/
+      - mkdir build
+      - cmake -S . -B build -DCMAKE_PREFIX_PATH=/app -DCMAKE_INSTALL_PREFIX=/app -DCatch2_DIR=/app -DPYTHON_EXECUTABLE=/usr/bin/python3 -DPYTHON_INCLUDE_DIR=/usr/include/python3.12 -DRDK_BUILD_CAIRO_SUPPORT=ON -DRDK_INSTALL_INTREE=OFF -DRDK_INSTALL_COMIC_FONTS=OFF -DRDK_BUILD_MAEPARSER_SUPPORT=OFF
+      - cd build && make -j5 && make install
+
+  - name: coot
+    sources:
+      - type: archive
+        url: https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/source/releases/coot-1.1.10.tar.gz
+        sha256: 447caa7bdd6f87b738d20f8db15aa278476c308e22506004ed145e04f85e0413
+      - type: file
+        path: io.github.pemsley.coot.metainfo.xml
+      - type: file
+        path: io.github.pemsley.coot.desktop
+    buildsystem: autotools
+    config-opts:
+      - --with-fftw-prefix=/app
+      - --with-boost=/app
+      - --with-boost-libdir=/app/lib
+      - --with-rdkit-prefix=/app
+      - --with-enhanced-ligand-tools
+      - --with-guile
+      - PYTHON=python3
+      - SHELL=/bin/bash
+    build-options:
+      append-path:
+        - /app/lib
+        - /app/bin
+      environment:
+        CXXFLAGS: "-g -O -fPIC"
+        CPPFLAGS: "-D_USE_BOOST=1"
+        LDFLAGS: "-L/app/lib -L/app/lib64 -L/usr/lib -lpython3.12"
+        LD_LIBRARY_PATH: "/app/lib64:/app/lib:/usr/lib"
+    post-install:
+      - install -Dm644 pixmaps/icons/hicolor_apps_scalable_coot.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    cleanup:
+      - /usr/lib/python*/site-packages/__pycache__
+      - /lib/pkgconfig
+      - /share/aclocal
+      - /share/doc
+      - /share/info
+      - /share/man
+      - /app/share/man
+      - /app/share/doc
+      - /app/lib/*.a
+      - /app/lib/*.la
+      - /app/tmp
```

Comment:
I use `/app/tmp` for storing necessary files temporarily. It's not typo.

Reply:
1. it is. 
2. I mentionned elsewhere in the review you shouldn't. Just build/install normally. A build only module can be removed with a `cleanup` rule that contain `'*'`.

Reply:
just in case you missed it in the documentation, cleanup rule are always *from* the FLATPAK_DEST. So this rule would expand to `/app/app/tmp`. Which is incorrect.